client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -415,10 +417,13 @@ class Annotation(Model):
         confidence_precision (float): Describe the confidence level of the annotation. Precision is defined as the % of annotation objects being true positive
         confidence_recall (float): Describe the confidence level of the annotation. Recall is defined as the % of true positives being annotated correctly
         deposition_date (date): Date when an annotation set is initially received by the Data Portal.
+        deposition_id (int): If the annotation is part of a deposition, the related deposition's id
+        files (list[AnnotationFile]): An array relationship with the files of this annotation
```

Comment:
@kandarpksk  Added the documentation to map the relationship between Annotation and AnnotationFiles.